### ISSUE TITLE
Camera Parameter

### ISSUE THREAD

**Issue by tanndx17** [edvardHua/PoseEstimationForMobile#10]
Hi, thanks for the great work. I can successfully load the apk and start to run on my snapdragon 845 android phone. Does this 2D estimation apk need camera parameter (width, height) from the phone?  I did not find any parameters in within the code

**Comment by edvardHua** [comment]
Of course, you can modify the `openCamera` function in `Camera2BasicFragment.kt` file and other corresponding code.
